File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_9.py
print(f'Tabela de preços:\n'
      'COD 1 - À vista(em espécie)           | 15% de desconto\n'
      'COD 2 - Cartão de débido              | 10% de desconto\n'
      'COD 3 - Cartão de Crédito(vencimento) | 5% de desconto\n')

preco_total = float(input('Digite o preço total da venda:'))
forma_de_pagamento = int(input('Digite o código do Pagamento:'))

if forma_de_pagamento == 1:
    print(f'Valor final a ser pago:{preco_total-(preco_total*(15/100))}')
elif forma_de_pagamento == 2:
    print(f'Valor final a ser pago:{preco_total-(preco_total*(10/100))}')
elif forma_de_pagamento == 3:
    print(f'Valor final a ser pago:{preco_total-(preco_total*(5/100))}')
else:
    print('Forma de pagamento inválido!')

File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-9.js
const prompt = require('prompt-sync')()

let preco = prompt('Digite o preço total da venda: ')

console.log(`
COD: 1 | À vista (em espécie)           |   15% |
COD: 2 | Cartão de débito               |   10% |
COD: 3 | Cartão de crédito (vencimento) |   5%  |`)

const codigo = Number(prompt('Digite o código da forma de pagamento: '))

switch(codigo){
    case 1:
        console.log(`Preço final:${preco * (1 - 15/100)}`)
        return
    case 2:
        console.log(`Preço final:${preco * (1 - 10/100)}`)
        return
    case 3:
        console.log(`Preço final:${preco * (1 - 5/100)}`)
        return
    default:
        console.log('Código inválido.')
        return
}
